database_manager/src/lib.rs

```diff
@@ -57,6 +60,13 @@ pub fn inspect_cli_app<'a, 'b>() -> App<'a, 'b> {
                 .default_value("sizes")
                 .possible_values(InspectTarget::VARIANTS),
         )
+        .arg(
+            Arg::with_name("output-dir")
+                .long("output-dir")
+                .value_name("DIR")
+                .help("Base directory for the output files. Defaults to the current directory")
```

Comment:
This feature currently doesn't create directories and requires it to exist. I could add the code to create directories, but would require changing the `inspect_db` function to return `Result<(), String>` instead of `Result<(), Error>`, or just print the error and return `()` from the function - not very nice as it's a failure scenario. I prefer the former if we do add this in.

Reply:
as we chatted about offline I think creating the outdir if it doesn't exist is reasonable

Reply:
Done!